### ISSUE TITLE
关于双线性汇合特征的规范化

### ISSUE THREAD

**Issue by dongburen** [HaoMood/bilinear-cnn#20]
请问torch.sqrt与F.normalize的作用分别是什么？
谢谢。
